var/www/html/memcached-admin-config.php.ctmpl

```diff
@@ -0,0 +1,31 @@
+<?php
+return array (
+  'stats_api' => 'Server',
+  'slabs_api' => 'Server',
+  'items_api' => 'Server',
+  'get_api' => 'Server',
+  'set_api' => 'Server',
+  'delete_api' => 'Server',
+  'flush_all_api' => 'Server',
+  'connection_timeout' => '1',
+  'max_item_dump' => '100',
+  'refresh_rate' => 5,
+  'memory_alert' => '80',
+  'hit_rate_alert' => '90',
+  'eviction_alert' => '0',
+  'file_path' => 'Temp/',
+  'servers' =>
+  {{ if service "memcached" }}
```

Comment:
Do we need to memcached templates, or can the two config files get data from the same template?

There's some cost on Consul when generating the templates, so it's generally good to reduce the number of templates being generated.